cumulusci/cli/cci.py

```diff
@@ -136,34 +137,41 @@ def show_debug_info():
     pdb.post_mortem()
 
 
-@click.group("main", help="")
-def cli():
-    """Top-level `click` command group."""
-
-
-@cli.command(name="version", help="Print the current version of CumulusCI")
-def version():
-    click.echo("CumulusCI version: ", nl=False)
-    click.echo(click.style(cumulusci.__version__, bold=True), nl=False)
-    click.echo(f" ({sys.argv[0]})")
-    click.echo(f"Python version: {sys.version.split()[0]}", nl=False)
-    click.echo(f" ({sys.executable})")
+def show_version_info():
+    console = rich.get_console()
+    console.print(
+        f"[blue bold]CumulusCI version: [blink] {cumulusci.__version__}[/] ({sys.argv[0]})"
+    )
+    console.print(
+        f"[purple bold]Python version: {sys.version.split()[0]} ({sys.executable})"
+    )
+    console.print()
 
-    click.echo()
     current_version = get_installed_version()
     latest_version = get_latest_final_version()
     if latest_version > current_version:
-        click.echo(
-            f"There is a newer version of CumulusCI available ({str(latest_version)})."
+        console.print(
+            f"[yellow flashing]There is a newer version of CumulusCI available ({str(latest_version)})."
```

Comment:
We should avoid flashing text:

> Flashing is generally a bad idea. It can cause all sorts of issues, from seizures to motion sickness. If you absolutely must have a flashing element there are a few things to consider.

https://accessibility.18f.gov/flashing/

```suggestion
            f"[yellow]There is a newer version of CumulusCI available ({str(latest_version)})."
```

---

💭: we'll want to set up shared styles so warnings and errors are consistent across commands.

Reply:
I don't think `flashing` was valid console markup anyway 😂 .